runtime/testy/studentAPI.py

```diff
@@ -75,7 +80,7 @@ def get_value(self, name, gamepad_number=0):
 
 class Robot(StudentAPI):
   deviceType_to_validParams = {
-    10 : ["duty_cycle", "enable"], 
+    10 : ["duty_cycle", "enable", "enc_pos", "enc_vel", "motor_current"],
```

Comment:
The problem is that we only want to have certain params as being available to the student, so we would have to filter the json file anyways.